### PR TITLE
Remove the DS project list toggle

### PR DESCRIPTION
**Pull request by ppadti**
Closes: [RHOAIENG-9576](https://issues.redhat.com/browse/RHOAIENG-9576)

## Description
This PR aims to remove the DS project list toggle.

Screenshot from 2024-07-22 14-31-34

## How Has This Been Tested?
In DS list page, we will not be having a toggle to switch projects.

## Test Impact
Updated cypress test.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by ppadti** [comment]
@kywalker-rh @xianli123 could you please take a look the UI changes?
One quick question, In project list page we had a image beside DS project name. I removed it in this PR. I just wanted to know that Is it fine to have list of project without that image or you prefer to have it for all the projects. Let me know your thoughts on this.

screenshot of the image I am talking about for ref:

Screenshot from 2024-07-19 18-21-47

File: frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx [hunk 1]
```diff
@@ -60,15 +59,6 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
...
-                    {project.metadata.labels?.[KnownLabels.DASHBOARD_RESOURCE] && (
-                      <FlexItem style={{ display: 'flex' }} data-testid="ds-project-image">
-                        <img
-                          style={{ height: 24 }}
-                          src={typedObjectImage(ProjectObjectType.project)}
-                          alt=""
-                        />
-                      </FlexItem>
-                    )}
                     <FlexItem>
```
Comment (andrewballantyne) [inline frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx:62]: I think you can drop the flex, no?

**Review by andrewballantyne — CHANGES_REQUESTED** [review]

**Comment by andrewballantyne** [comment]
> @kywalker-rh @xianli123 could you please take a look the UI changes? One quick question, In project list page we had a image beside DS project name. I removed it in this PR. I just wanted to know that Is it fine to have list of project without that image or you prefer to have it for all the projects. Let me know your thoughts on this.
> 
> screenshot of the image I am talking about for ref:
> 
> Screenshot from 2024-07-19 18-21-47

I'll step in here 🙂 Yes, we do need to get rid of the identifier. We are looking to get so far away from this concept of "Data Science Projects are different" that we are even RFE-ing a rename of this page to just `Projects`. So definitely get rid of it 👍

**Review by mturley — APPROVED** [review]
LGTM and tested locally! Awesome to finally have this weird toggle gone.

**Review by gitdallas — APPROVED** [review]

**Pull request merged**